extension/lib/errors/AuthFailedError.js

```diff
@@ -0,0 +1,11 @@
+const EAUTHFAILED = 'EAUTHFAILED'
+
+class AuthFailedError extends Error {
```

Comment:
To me it would be nice if customer authorisation errors would go to ./extension/bigcommerce/customer/auth

Reply:
Wouldn't it be nicer to have all errors grouped in an "errors" folder?

Reply:
This is similar as repository grouping comment - we should talk about this at least that you get my point of view. I've seen others are grouping it like you're suggesting - the thing is that I believe dependencies should point inwards to the extent it is possible. Let's talk when we're both here.

Reply:
done